mappings/net/minecraft/client/settings/GameSettings.mapping

```diff
@@ -40,7 +45,36 @@ CLASS none/bdt net/minecraft/client/settings/GameSettings
 		FIELD x VBO Lnone/bdt$a;
 		FIELD y TOUCHSCREEN Lnone/bdt$a;
 		FIELD z CHAT_SCALE Lnone/bdt$a;
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;ZZ)V
+			ARG 0 key
+			ARG 1 isSlider
+			ARG 2 isBoolean
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;ZZFFF)V
+			ARG 0 key
+			ARG 1 isSlider
+			ARG 2 isBoolean
+			ARG 3 sliderMin
+			ARG 4 sliderMax
+			ARG 5 sliderStep
+		METHOD a isSlider ()Z
+		METHOD a setMaximum (F)V
+			ARG 0 value
+		METHOD a getOption (I)Lnone/bdt$a;
+			ARG 0 id
+		METHOD b isBoolean ()Z
```

Comment:
I've changed it to `isToggle` but it's still very confusing without documentation, both of the values (`toggle` and `slider`) are just booleans. They indicate whether or not the option is a numeric slider value option or a boolean value option.